File: exer4.java
//ADRIEL XAVIER PEREIRA DA SILVA
//ARTHUR RODRIGUES PORTELA
//DENISE XAVIER DE ALMEIDA

import java.util.Scanner;


public class exer4 {
public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);
    System.out.println("Digite a quantidade de até 4 vendas:");
    String vendasProdutos = scanner.nextLine();

    System.out.println("Digite o preço da venda 1:");
    String precoProdutos = scanner.nextLine();
       
    System.out.println("Digite o preço da venda 2:");
    String precoProdutos1 = scanner.nextLine();

    System.out.println("Digite o preço da venda 3:");
    String precoProdutos2 = scanner.nextLine();    
  
    System.out.println("Digite o preço da venda 4:");
    String precoProdutos3 = scanner.nextLine();
    
    int p1 = Integer.parseInt(precoProdutos);
    int p2 = Integer.parseInt(precoProdutos1);
    int p3 = Integer.parseInt(precoProdutos2);
    int p4 = Integer.parseInt(precoProdutos3);

    int soma =p1 + p2 + p3 + p4;
        if (soma >= 120){
            String resultado = String.format( "A quantidade de vendas foi de %s. O preço total de %d", vendasProdutos, soma);
            System.out.println(resultado);
        }
        else{
            int frete = soma + 20;
            String freteSoma = String.valueOf(frete);
            String resultado = String.format("A quantidade de vendas foi de %s. O preço total + frete foi de %s",vendasProdutos, freteSoma);
            System.out.println(resultado);
        }

    

        scanner.close();
}
}

File: exer4.js
//ARTHUR RODRIGUES PÓRTELA
//DENISE XAVIER DE ALEMIDA
//ADRIEL XAVIER PEREIRA DA SILVA
//LUIS FELIPE SANTOS RIZZARO
/*
4. Peça para o usuário digitar o número de produtos vendidos, o preço unitário de cada produto e o valor do frete e forneça o custo total da venda.

usuario escolher produtos >/ fazer variaveis com produto e preço >/

somar tudo > caso o valor passe de tal valor frete grátis ou acrescenta 
valor de 20 reis > somar a quantidade de produtos vendidos. */
let resul = document.getElementById("resultado")
let vendas = prompt("Digite a quantidade de vendas, até 5 unidades:");
let preco_1 = prompt("Venda 1:") * 1; 
let preco_2 = prompt("Venda 2:") * 1; 
let preco_3 = prompt("Venda 3:") * 1;
let preco_4 = prompt("Venda 4:") * 1;
let preco_5 = prompt("Venda 5:") * 1;

function calculoFrete(){
    let soma = preco_1 + preco_2 + preco_3 + preco_4 + preco_5;
        if ( soma >= 120){
            return alert(`O valor da compra foi de ${soma} reais.`);
        } else{
            let frete = soma + 15
            return alert(`O valor da compra foi de ${frete}, por conta do frete.`);
        }
}
calculoFrete();

